--- app/src/main/java/com/example/data/WebsiteRepository.kt
+++ app/src/main/java/com/example/data/WebsiteRepository.kt
@@ -14,10 +14,10 @@
         ),
         "JAP + ENG + SUB" to listOf(
-            WebsiteModel("Yeanime", "https://yenime.net/"),
             WebsiteModel("Hianime", "https://hianime.ro/home"),
-            WebsiteModel("Animeflix", "https://animeflix.app/"),
-            WebsiteModel("Anistream", "https://anistream.one/home")
+            WebsiteModel("Yeanime", "https://yenime.net/"),
+            WebsiteModel("Anistream", "https://anistream.one/home"),
+            WebsiteModel("Animeflix", "https://animeflix.app/")
         ),
         "JAP + SUB" to listOf(
